mmv1/products/backupdr/BackupVault.yaml

```diff
@@ -198,3 +198,15 @@ properties:
     description: "Optional. User annotations. See https://google.aip.dev/128#annotations\nStores
       small amounts of arbitrary data. "
     min_version: 'beta'
+  - name: 'accessRestriction'
```

Comment:
Is this field updatable? If so, please include it in an update test and update the value.

Reply:
This field is not updatable - it's only used during create backup vault step.

Reply:
It should be `immutable: true` then